torchinductor/config.py

```diff
@@ -49,6 +49,12 @@
 # replace small reductions with pointwise, disable with `= 1`
 unroll_reductions_threshold = 8
 
+# Cudagraphify will run even if we have graph mutations
+always_attempt_cudagraphify = True
```

Comment:
Just for my education? Why have this flag? Is this to prefetch other problems?

Reply:
There are cases in prod models where we over-enthusiastically decide that a graph was mutated and then abort attempting cudagraphify. This just lets it always have a go.